runtime/runtime/src/lib.rs

```diff
@@ -1330,6 +1330,17 @@ impl Runtime {
             vec![]
         };
 
+        // Remove testnet account with large storage key.
+        if ProtocolFeature::RemoveAccountWithLongStorageKey.protocol_version() == protocol_version
```

Comment:
can we also restrict this to testnet?

Reply:
That's a hard question. I want to avoid using chain_id where possible because this is kind of a reversed dependency from runtime on chain, while it should be the opposite.
Mainnet doesn't have `contractregistry.testnet` account and is not going to have it.

Reply:
But there migration doesn't need to go through runtime, does it? Could we do it as part of a database migration during node startup? We just need to construct a `TrieUpdate` from `Store` and then remove the account, where we can check whether `chain_id == testnet`

Reply:
But it changes state root, so it must be a part of some verified state transition. It must be replayable.